CrewChiefV4/SpeechRecogniser.cs

```diff
@@ -419,6 +422,26 @@ public void initialiseSpeechEngine()
                         break;
                     }
                 }
+                foreach (String s in PIT_STOP_ADD)
```

Comment:
should we make this check that the game type is iRacing? It's reducing SRE accuracy for no benefit on other games